cypress/integration/SignInTests.spec.js

```diff
@@ -0,0 +1,51 @@
+/// <reference types="cypress" />
+
+
+import { AuthenticationPage } from "../PageObjects/AuthenticationPage";
+import { myAccountPage } from "../PageObjects/myAccountPage";
+import { PageHeader } from "../PageObjects/PageHeader";
+
+const authPage = new AuthenticationPage()
+const header = new PageHeader()
+const myAccPage = new myAccountPage()
+
+let currentUser //placeholder for currently loaded Fixture User data
+
+describe('Login Tests with Fixtures Data', () => {
+
+    beforeEach(function () {
```

Comment:
do we need to export user fixture before each test?

Reply:
no, we can use before function to export user fixture once.

Reply:
please do